ecl/hqlcpp/hqliproj.cpp

```diff
@@ -2824,9 +2825,11 @@ IHqlExpression * ImplicitProjectTransformer::createTransformed(IHqlExpression *
                 getTransformedChildren(expr, args);
 
                 IHqlExpression * transform = &args.item(transformPos);
+                // NB: transform may be destroyed before this is tested
```

Comment:
When I first read this, I thought the comment implied that transform may have been destroyed by the time line 2829 is executed.  May be "transform may be destroyed **after next line** ..."?